content/exploratory_notebooks/Centrality.md

```diff
@@ -0,0 +1,462 @@
+---
+jupyter:
+  kernelspec:
+    display_name: Python 3 (ipykernel)
+    language: python
+    name: python3
+  language_info:
+    codemirror_mode:
+      name: ipython
+      version: 3
+    file_extension: .py
+    mimetype: text/x-python
+    name: python
+    nbconvert_exporter: python
+    pygments_lexer: ipython3
+    version: 3.8.10
+  nbformat: 4
+  nbformat_minor: 5
+---
+
+::: {.cell .markdown}
+## Centrality Algorithms
+:::
+
+::: {.cell .code execution_count="207"}
+``` {.python}
+import networkx as nx
+import matplotlib.pyplot as plt
+import random as ran
+import numpy as np
+import seaborn as sns
+import pandas as pd
+```
+:::
+
+::: {.cell .code execution_count="4"}
+``` {.python}
```

Comment:
How are you converting your notebooks to markdown?

These blocks of code shouldn't be there.